app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -175,6 +175,9 @@ def parse_openshift_image(openshift_image)
         ref = "#{ContainerImage::DOCKER_PULLABLE_PREFIX}#{id}"
         new_result = parse_container_image(id, ref)
 
+        #new_result[:ems_ref] = openshift_image.metadata.uid
```

Comment:
@simon3z No, I commented this out because of https://github.com/ManageIQ/manageiq-schema/pull/21#issuecomment-312681461

Do you think we should remove this column or put the digest there? Anything else we can do?